docs/index.md

```diff
@@ -423,6 +423,253 @@ You can monitor the progress via the SageMaker UI console. Here is an example of
 
 ## Monitor ML Models in Production
 
+## Superwise.ai
+
+Superwise provide organization to streamline the model observability and monitoring process 🚀
+It's platform agnostic and provide an extensive freemium tier.
+
+In this part, you'll integrate with [Superwise.ai](https://www.superwise.ai/) gain full visibility on your deployed models. 
```

Comment:
Missing `to` -> `integrate with [Superwise.ai](https://www.superwise.ai/) to gain full visibility`

Reply:
fixed